config/manifests/bases/neutron-operator.clusterserviceversion.yaml

```diff
@@ -18,6 +18,10 @@ spec:
       displayName: Neutron API
       kind: NeutronAPI
       name: neutronapis.neutron.openstack.org
+      specDescriptors:
```

Comment:
im not sure why this got here, I would assume when this was last updated to the most recent mariadb this wasn't built (is this "make bundle"?)